boxsdk/util/translator.py

```diff
@@ -191,5 +193,28 @@ def translate(self, session, response_object):
 
         return translated_obj
 
+    def untranslate(self, value):
+        """
+        Untranslates a given object into a dictionary.
+
+        :param value:
+            The object to untranslate.
+        :rtype value:
+            `dict`
+        :return:
+            A dictionary containing the untranslated object.
+        """
+        if hasattr(value, '_response_object') and isinstance(value._response_object, dict):  # pylint:disable=protected-access
+            new_value = self.untranslate(value._response_object)  # pylint:disable=protected-access
```

Comment:
You can instead set `value = ` and get rid of the return statement. That way it'll fall-through to the `if isinstance(value, dict)`, rather than needing to do a recursion.